source/Ship.cpp

```diff
@@ -1519,7 +1519,7 @@ bool Ship::CanSendHail(const PlayerInfo &player, bool allowUntranslated) const
 		return false;
 
 	// Make sure this ship is able to send a hail.
```

Comment:
Update comment pls

Reply:
Something like this?
```suggestion
	// Make sure this ship is able to send a hail message, unless it has the quiet or mute personality.
```

Reply:
ye, also watch the length of the line, but I think it is fine